snippets/views_embedded_ruby.cson

```diff
@@ -27,6 +27,34 @@
       <% end
     """
 
+'.text.haml > .line.ruby.haml > .source.ruby':
+  'haml embedded each':
+    'prefix': '.each'
+    'body': """
+      .each do |${1:member}|
+        = ${1:member} $2
+    """
+
+  'haml embedded block':
+    'prefix': 'do'
+    'body': """
+      do
+        = $1
+    """
+
+  'haml embedded block with parm':
+    'prefix': 'dop'
+    'body': """
+      do |${1:member}|
+        = ${1:member}$2
+    """
+
+'.text.html.erb > .embedded.line.erb > .source.ruby,
```

Comment:
Selector for both erb and haml, these snippets don't use `<% %>` tags so You can use with haml